cmd/kops/validate_cluster.go

```diff
@@ -94,37 +96,38 @@ func RunValidateCluster(f *util.Factory, cmd *cobra.Command, args []string, out
 		clientcmd.NewDefaultClientConfigLoadingRules(),
 		&clientcmd.ConfigOverrides{CurrentContext: contextName}).ClientConfig()
 	if err != nil {
-		return fmt.Errorf("cannot load kubecfg settings for %q: %v", contextName, err)
+		return fmt.Errorf("Cannot load kubecfg settings for %q: %v\n", contextName, err)
```

Comment:
FYI, the advice is to start with lower cases, so that errors can be chained.

This advice will change once go gets exceptions, of course ;-)